<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>301</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="load_profileLabel">
       <property name="text">
        <string>Existing profiles</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="load_profileComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="profile_nameLabel">
       <property name="text">
        <string>Profile name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="profile_nameLineEdit">
       <property name="toolTip">
        <string>Max 11 Characters!</string>
       </property>
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="min_rpmLabel">
       <property name="text">
        <string>MINrpm</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="min_rpmLineEdit">
       <property name="toolTip">
        <string> Range: 0-200, MINrpm &lt;= MAXrpm - 5 !!!!!!!!</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
       <property name="placeholderText">
        <string>0 - 200</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="max_rpmLabel">
       <property name="text">
        <string>MAXrpm</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="max_rpmLineEdit">
       <property name="toolTip">
        <string> Range: 0-200, MINrpm &lt;= MAXrpm - 5 !!!!!!!!</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
       <property name="placeholderText">
        <string>0 - 200</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="filterComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="sample_numberLabel">
       <property name="text">
        <string>Sample number</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="sample_numberLineEdit">
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>0 - 20</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly|Qt::ImhFormattedNumbersOnly</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="placeholderText">
        <string>0 - 20</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="load_defaultPushButton">
       <property name="text">
        <string>Load default Profile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_profilePushButton">
       <property name="text">
        <string>Save / Update Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="load_profilePushButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_profilePushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
